DEPS

```diff
@@ -30,6 +30,13 @@ hooks = [
     'pattern': '.',
     'action': ['python', 'script/bootstrap.py'],
   },
+  {
+    # Download hermetic xcode for goma
+    'name': 'download_hermetic_xcode',
+    'pattern': '.',
+    'condition': 'host_os == "mac"',
+    'action': ['vpython3', 'build/mac/download_hermetic_xcode.py'],
```

Comment:
what about licensing issues? currently it will be available for everyone.

Reply:
This still requires you to accept the license agreement https://github.com/brave/brave-core/pull/12674/files#diff-f8754de7248ea1cafbdb9abb2a470e4479402a4e042f4713abf5289550eadbc2R123

Reply:
1. will it show the actual license or just asks for your password? if you're running everything as `root`, will you be prompted with anything?
2. will the `npm run sync/build` (non-goma build) still work with the system-installed Xcode if we don't run this script?

Reply:
This is settled for now by only download for brave goma users